x-pack/test/ui_capabilities/common/services/ui_capabilities.ts

```diff
@@ -80,25 +73,10 @@ export class UICapabilitiesService {
       );
     }
 
-    const dom = cheerio.load(response.data.toString());
-    const element = dom('kbn-injected-metadata');
-    if (!element) {
-      throw new Error('Unable to find "kbn-injected-metadata" element ');
-    }
-
-    const dataAttrJson = element.attr('data');
-
-    try {
-      const dataAttr = JSON.parse(dataAttrJson);
-      return {
-        success: true,
-        value: dataAttr.vars.uiCapabilities as UICapabilities,
-      };
-    } catch (err) {
-      throw new Error(
-        `Unable to parse JSON from the kbn-injected-metadata data attribute: ${dataAttrJson}`
-      );
-    }
+    return {
```

Comment:
Now that we're no longer parsing the HTML returned for this data, we should be able to add

https://github.com/elastic/kibana/blob/c41010760e636a4882c4b73d31de20aaa4866a0b/x-pack/test/api_integration/config.js#L62

to 

https://github.com/elastic/kibana/blob/c41010760e636a4882c4b73d31de20aaa4866a0b/x-pack/test/ui_capabilities/common/config.ts#L49

and get faster CI runs!!!